packages/terra-date-time-picker/src/terra-dev-site/test/date-time-picker/DateTimePicker12HourMobile.test.jsx

```diff
@@ -34,7 +36,7 @@ export default class DateTimePicker12HourMobile extends React.Component {
 
   render() {
     return (
-      <div style={removeInputCursorForScreenshots}>
+      <div className={cx('date-time-picker-input-cursor')}>
```

Comment:
This is not a cursor. Such wrappers could simply be `${component-name}-wrapper` imo.

Reply:
Changed the class name to a more appropriate 'content-wrapper'